restartmanager/restartmanager_test.go

```diff
@@ -9,26 +9,28 @@ import (
 
 func TestRestartManagerTimeout(t *testing.T) {
 	rm := New(container.RestartPolicy{Name: "always"}, 0).(*restartManager)
-	should, _, err := rm.ShouldRestart(0, false, 1*time.Second)
```

Comment:
I want to use a variable instead of expression. The function parameter is simple and it  is helpful to debug